web/packages/shared/services/loc/loc.ts

```diff
@@ -42,6 +44,20 @@ export function displayDate(date: Date) {
   }
 }
 
+// displayShortDate accepts a date and returns formatted as 'MM dd, yyyy'
+export function displayShortDate(date: Date) {
+  try {
+    if (isTest) {
+      return format(dateToUtc(date), cfg.shortFormat);
+    }
+    return format(date, cfg.shortFormat);
+  } catch (err) {
+    logger.error('displayDate()', err);
```

Comment:
maybe a nitpick, but this log message might be misleading?
```suggestion
    logger.error('displayShortDate()', err);
```